### PR TITLE
[#32] Implement unit testing of Controller layer

### PR DESCRIPTION
**Pull request by Sheptytskyid**

### CODE REVIEW

File: src/main/java/com/manardenza/TestUtils.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import java.util.Map;
+
+@Data
+public class TestUtils {
+
+    public static final String HOTEL_NAME = "Mariott";
+    public static final String CITY = "Kiev";
+    public static final String FIRST_NAME = "Name";
+    public static final String LAST_NAME = "Surname";
+    public static final Date RESERVED_FROM = new Date(117, 1, 10);
```
Comment (Sheptytskyid) [inline src/main/java/com/manardenza/TestUtils.java:19]: MINOR Remove this use of "Date"; it is deprecated. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
MINOR Make this member "protected". [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS2386)

File: src/main/java/com/manardenza/TestUtils.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    public static final Date RESERVED_TO = new Date(117, 1, 15);
```
Comment (Sheptytskyid) [inline src/main/java/com/manardenza/TestUtils.java:20]: MINOR Remove this use of "Date"; it is deprecated. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
MINOR Make this member "protected". [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS2386)

File: src/main/java/com/manardenza/TestUtils.java [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    public static final User USER = new User(FIRST_NAME, LAST_NAME);
+    public static final Room ROOM = new Room("Room", 2, 300);
+    public static final List<Room> ROOMS_LIST = new ArrayList<>();
```
Comment (Sheptytskyid) [inline src/main/java/com/manardenza/TestUtils.java:23]: MINOR Make this member "protected". [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS2386)

File: src/main/java/com/manardenza/TestUtils.java [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    public static final Hotel HOTEL = new Hotel(HOTEL_NAME, CITY, ROOMS_LIST);
+    public static final Map<String, List<Room>> ROOMS_MAP = new HashMap<>();
```
Comment (Sheptytskyid) [inline src/main/java/com/manardenza/TestUtils.java:25]: MINOR Make this member "protected". [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS2386)

File: src/main/java/com/manardenza/TestUtils.java [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+import lombok.Data;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Data
+public class TestUtils {
```
Comment (Sheptytskyid) [inline src/main/java/com/manardenza/TestUtils.java]: MAJOR Add a private constructor to hide the implicit public one. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS1118)

**Comment by Sheptytskyid** [comment]
SonarQube analysis reported 7 issues
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 6 minor

Watch the comments in this conversation to review them.

[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'
[MINOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR'

**Review by zozich — APPROVED** [review]
Ok

File: src/main/java/com/manardenza/TestUtils.java [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+import com.manardenza.entity.Room;
+import com.manardenza.entity.User;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class TestUtils {
```
Comment (Sheptytskyid) [inline src/main/java/com/manardenza/TestUtils.java]: MAJOR Add a private constructor to hide the implicit public one. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS1118)

File: src/main/java/com/manardenza/Main.java [hunk 7]
```diff
@@ -1,5 +1,6 @@
 package com.manardenza;
 
+import com.manardenza.utils.DatabaseSeeder;
 import org.slf4j.LoggerFactory;
 
 public class Main {
```
Comment (Sheptytskyid) [inline src/main/java/com/manardenza/Main.java:6]: MAJOR Add a private constructor to hide the implicit public one. [rule](https://sonarqube.com/coding_rules#rule_key=squid%3AS1118)

**Comment by Sheptytskyid** [comment]
SonarQube analysis reported 8 issues
* ![MAJOR][MAJOR] 2 major
* ![MINOR][MINOR] 6 minor

Watch the comments in this conversation to review them.

[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'
[MINOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR'

**Comment by Sheptytskyid** [comment]
SonarQube analysis reported 7 issues
* ![MAJOR][MAJOR] 1 major
* ![MINOR][MINOR] 6 minor

Watch the comments in this conversation to review them.

[MAJOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR'
[MINOR]: https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR'

**Pull request merged**